bin.js

```diff
@@ -208,9 +208,14 @@ if (result !== null) {
 function runTool (args, Tool, version) {
   const onPort = args['on-port']
 
+  if (!onPort) {
+    console.log('To generate the report press: Ctrl + C')
+  }
+
   const tool = new Tool({
     sampleInterval: parseInt(args['sample-interval'], 10),
-    detectPort: !!onPort
+    detectPort: !!onPort,
+    debug: 'debug' in args
```

Comment:
this is using miminist, so you can set `debug` as a boolean there, with a default value of `false`.
Unfortunately this would enable debug with `--debug false` , which would not be perfect.